csv2rdf/thesession/mapping.json

```diff
@@ -37,13 +36,13 @@
     "tuneset_id": "https://thesession.org/tuneset_id",
     "username": "http://www.wikidata.org/prop/direct/P554",
     "setting_id": "https://thesession.org/setting_id",
-    "type": "http://www.wikidata.org/prop/direct/P2308",
+    "type": "http://www.wikidata.org/prop/direct/P136",
     "type_wiki": "http://www.wikidata.org/prop/direct/P2308",
     "meter": "http://www.wikidata.org/prop/direct/P3440",
     "meter_wiki": "http://www.wikidata.org/prop/direct/P3440",
     "mode": "http://www.wikidata.org/prop/direct/P826",
     "mode_wiki": "http://www.wikidata.org/prop/direct/P826",
-    "tunebooks": "http://www.wikidata.org/entity/Q66382988",
+    "tunebooks": "https://thesession.org/tunebooks",
```

Comment:
I don't think this is correct. This should be a predicate, right?

Reply:
It is a predicate, that's why I changed the wiki Q value.